R/format_dhs_utils.R

```diff
@@ -98,7 +106,24 @@ get_births <- function(dat,
   
   whichnotdied <- which(datnew$died == 0)
   # EDIT - fix this so that the interval ending with Inf doesn't have a closed right bracket
-  datnew$age_interval <- cut(datnew$age_at_censoring, breaks = c(0,month.cut), include.lowest = TRUE, right = FALSE)
+  datnew$age_interval <- as.character(cut(datnew$age_at_censoring, breaks = c(0,month.cut), include.lowest = TRUE, right = FALSE))
```

Comment:
Do you have a problem with this being a character instead of a factor? Makes it easier to add new values later.